spring-cloud-dataflow-rest-resource/src/main/java/org/springframework/cloud/dataflow/rest/SkipperStream.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2017 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.springframework.cloud.dataflow.rest;
+
+/**
+ *
+ * @author Ilayaperumal Gopinathan
+ */
+public class SkipperStream {
+
+	public static final String SKIPPER_STREAM_PREFIX = "scdf_";
```

Comment:
What about if we create a `DataFlowSkipperProperties` `@ConfigurationClass`  (prefix spring.cloud.dataflow.skipper) and allow this to be configurable?

Reply:
Actually, upon discussion, seems the principle of least surprise is to not have a prefix at all.  This also simplifies the status lookup delegated to skipper from scdf shell.